scripts/lib/CIME/case.py

```diff
@@ -1076,7 +1077,8 @@ def get_mpirun_cmd(self, job="case.run"):
         mpi_attribs = {
             "compiler" : self.get_value("COMPILER"),
             "mpilib"   : self.get_value("MPILIB"),
-            "threaded" : self.get_build_threaded()
+            "threaded" : self.get_build_threaded(),
+            "unit_testing" : False
```

Comment:
I see - I think that this logic is messed up a little - we should get the list of allowed attributes from the config_machines.xsd file and check against that - but that's another issue.